vizro-core/docs/pages/user-guides/custom-css.md

```diff
@@ -314,6 +314,14 @@ It's essential to understand the relationship between the targeted CSS class or
 
 ## Common examples
 
+### Make your CSS responsive to theme switches with variables
+To ensure your CSS adapts to theme changes, we recommend using CSS variables (`var`) whenever possible. For a
+comprehensive list of available variable names, refer to the
+[Bootstrap documentation](https://getbootstrap.com/docs/5.3/customize/css-variables/). Note that our
+Bootstrap stylesheet is still under development, so not all Bootstrap variables are currently available.
```

Comment:
Is there a list of which are or are not available? Would it help to offer one, or put it in the release notes?

Reply:
Noo, there is no list available and it would also take way too long to create one 😄 I am still hesitating whether to put that information in at all. If it's not complete, do you think we should not mention the Bootstrap variables for now?

Reply:
Would a potential user of the stylesheet be able to see from the code whether a variable is supported? Maybe we could just point them to the code so they can check before using. I think it is worth acknowledging support but pointing out the partial coverage, as you've done, so if that's as far as you can inform the user, it's fine. Just nice to help them out if we can.

Reply:
No, not really. There will be a color or value assigned but it might come from the default Bootstrap theme which will then look off inside vizro. So in general they can use all of these variables as there are for sure values assigned, but they might just look off inside vizro e.g. there might be some random blue elements 😄

Reply:
Let me add that in :)